packages/hierarchies/src/hierarchies/HierarchyProvider.ts

```diff
@@ -136,6 +136,12 @@ interface HierarchyProviderLocalizedStrings {
   other: string;
 }
 
+/**
+ * A path of hierarchy node identifiers for filtering the hierarchy with additional options.
+ * @beta
+ */
+export type FilteringPath = HierarchyNodeIdentifiersPath | { path: HierarchyNodeIdentifiersPath; options?: { autoExpand?: boolean } };
```

Comment:
I'd rename this to `HierarchyFilteringPath`

Reply:
Renamed.